msg/RobotState.msg

```diff
@@ -0,0 +1,8 @@
+int8 ERROR = -1
+int8 E_STOP = 0
+int8 STANDBY = 1
```

Comment:
What about `IDLE` or  `RUNNING`? Nevertheless, `STANDBY` is also ok ;)

Reply:
The animation of leds is `READY_ANIM_ID` so `READY` also sounds good.

Reply:
IMO STANDBY is the most precise.
![image](https://github.com/user-attachments/assets/d764339d-4c7f-4858-b13c-6292aa6af2e5)

Reply:
Additional states could be: SLEEP, AUTONOMY, TELEOP.